magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -149,20 +150,19 @@ public open class NavigationDelegate(
     navigationPropagator.onNavigatedTo(currentNavigable)
     when (currentState) {
       is LifecycleState.Shown, is LifecycleState.Resumed -> {
-        containerView!!.addView(
-          currentNavigable.view!!,
-          direction.indexToAddView(containerView!!)
-        )
+        val templatedView = templateApplier?.onViewCreated(currentState.context!!, currentNavigable.view!!) ?: currentNavigable.view!!
```

Comment:
this allows each navigator to customize UI for each Navigable it puts into the containerView. This is separate from navigator-level UI - to do that, one should continue to define UI in their Journey's XML.